sdk/textanalytics/ai-text-analytics/package.json

```diff
@@ -70,8 +70,8 @@
     "integration-test:browser": "dev-tool run test:browser",
     "integration-test:node": "dev-tool run test:node-js-input -- --timeout 5000000 \"dist-esm/test/**/*.spec.js\"",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
-    "lint": "eslint package.json api-extractor.json src test --ext .ts",
+    "lint:fix": "eslint README.md package.json api-extractor.json src test --ext .ts,.md,.javascript,.js --fix --fix-type [problem,suggestion]",
+    "lint": "eslint README.md package.json api-extractor.json src test --ext .ts,.md,.javascript,.js",
```

Comment:
curious whether `.md,.javascript,.js` is required in `--ext`.  Previously we could lint package.json without specifying `.json` extension

Reply:
I tried to experiment with different `--ext` arguments. It turns out that the eslint in only linting the snippet when either
- `--ext` is not defined at all. It is because ESLint v7 [automatically lints file extensions specified in overrides[].files patterns in config files](https://github.com/eslint/rfcs/blob/0253e3a95511c65d622eaa387eb73f824249b467/designs/2019-additional-lint-targets/README.md). It looks like eslint encourages not using `--ext` options when linting a wide array of file types as it is redundant with the `override` property.
- `--ext .md,.javascript` is included.

This also brings up another question where we might only want to keep `.javascript` instead of keeping both `.javascript` and `.js`, but only with the promise that all javascript sinppets in `README.md` are named `javascript`. I tried to look over multiple `README.md` and this turns out to be true, but I am still unsure whether it holds true for the whole repo.

Reply:
we discussed and decided to remove `--ext`